media-libs/avidemux-core/files/avidemux-core-2.6.8-gcc6.patch

```diff
@@ -0,0 +1,50 @@
+--- a/avidemux_core/ADM_coreUtils/src/prefs.cpp
++++ b/avidemux_core/ADM_coreUtils/src/prefs.cpp
+@@ -44,8 +44,8 @@
+    const char *name2;
+    ADM_paramType  type;
+    const char *defaultValue;
+-   float    min;
+-   float    max;
++   double    min;
++   double    max;
```

Comment:
Taken from part of upstream commit https://github.com/mean00/avidemux2/commit/b54303bc9b9310e288a1dde9c923f48e854bd55f